frontend/package.json

```diff
@@ -117,6 +117,7 @@
     "unist-util-visit": "^4.1.2",
     "vega": "^5.22.1",
     "vega-embed": "^6.21.0",
+    "vega-functions": "^5.13.1",
```

Comment:
I think this should probably be handled in the `resolutions` section of `package.json` (vega-functions is not a direct dependency of ours - it's a transitive dependency and we want to just specify that we don't get an outdated version of it - we don't want to "promote" it to a direct dependency.)

Reply:
Agree with Tim here, transitive dependencies causing security vulnerabilities should be handled in `resolutions` section.